src/checks/herokuLogDrain.check.js

```diff
@@ -0,0 +1,123 @@
+const fetch = require('node-fetch');
+const logger = require('@financial-times/n-logger').default;
+const Check = require('./check');
+const status = require('./status');
+
+const defaultPanicGuide = '[Check whether the app has been migrated to use log drains](https://financialtimes.atlassian.net/wiki/spaces/DS/pages/7883555001/Migrating+an+app+to+Heroku+log+drains). If it has been migrated then the log drain is either misconfigured or missing, follow the migration guide to correct this.';
```

Comment:
I'm not sure if Heimdall will parse this Markdown? If so, cool! If not it's still a nice way of including the link 👍

Reply:
I don't know if Heimdall will parse it either, I used Markdown based on the [check spec saying `string (markdown)` for this field](https://docs.google.com/document/d/1ftlkDj1SUXvKvKJGvoMoF1GnSUInCNPnNGomqTpJaFk/edit#heading=h.3zknx8k6ja6i). Happy for us to try it and find out?

Reply:
Oh cool! Yeah I would love it if this works. Lots of checks could use a bit of formatting to make them easier to read 👍